torch/nn/utils/fusion.py

```diff
@@ -26,3 +26,23 @@ def fuse_conv_bn_weights(conv_w, conv_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
     conv_b = (conv_b - bn_rm) * bn_var_rsqrt * bn_w + bn_b
 
     return torch.nn.Parameter(conv_w), torch.nn.Parameter(conv_b)
+
+def fuse_linear_bn_eval(linear, bn):
+    assert(not (linear.training or bn.training)), "Fusion only for eval!"
+    fused_linear = copy.deepcopy(linear)
+
+    fused_linear.weight, fused_linear.bias = fuse_linear_bn_weights(
+        fused_linear.weight, fused_linear.bias,
+        bn.running_mean, bn.running_var, bn.eps, bn.weight, bn.bias)
+
+    return fused_linear
+
+def fuse_linear_bn_weights(linear_w, linear_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
+    if linear_b is None:
+        linear_b = bn_rm.new_zeros(bn_rm.shape)
```

Comment:
should this be just `linear_b = torch.zeros_like(bn_rm)`?

Reply:
thanks, fixed